daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -97,6 +98,9 @@ def enable(name=None):
     else:
         for key in _mapping:
             do_patch(key)
+    if verbose and sys.stderr is not None:
+        sys.stderr.write("Intel DAAL solvers for sklearn enabled: "
+                         "https://intelpython.github.io/daal4py/sklearn.html\n")
```

Comment:
We probably need a message that accounts for selective patching, e.g. if only some algorithms have been patched. Might be confusing otherwise.

Reply:
I wanted to do that initially (e.g. one line for each class / function patched) but I was it afraid it would maybe be too verbose. The information about the class that are patched is available by clicking on the link.

@oleksandr-pavlyk what's your opinion?

Reply:
@ogrisel I meant to somehow indicate which of all the algos were patched if name!=None. The message is ok if name==None.